highfidelity/hifi
# Case 21757: show up audio screen using Settings > Audio

# TEST PLAN
- Open Interface in desktop mode
- Open audio settings screen using the menu with `Settings > Audio`
- Screen should be visually/functionally the same as opening through the audio app in the toolbar
- Switch to HMD mode
- Open tablet
- Open audio settings screen by clicking through `Menu > Settings > Audio`
- Screen should be visually/functionally the same as opening through the audio app and desktop mode `Settings > Audio` screen through the menu

https://highfidelity.manuscript.com/f/cases/21757/RC81-Audio-Settings-window-accessed-by-Settings-Audio-is-blank
